pom.xml

```diff
@@ -72,7 +72,7 @@
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
 
     <!-- dependency versions -->
-    <jackson.version>2.13.4</jackson.version>
+    <jackson.version>2.14.0-rc1</jackson.version>
```

Comment:
How about 2.13.4.2 instead? I'd rather avoid relying on a RC.

Reply:
Yes, 2.13.4.2 is definitely better and includes the fix according to https://github.com/FasterXML/jackson-databind/issues/3590